### DISCUSSION TITLE
trying to get SSL on my synology web server

### DISCUSSION THREAD

**Discussion by mackconsult** [acmesh-official/acme.sh#6120] [Q&A]
I followed the directions on this but for some reason it errors out:

[https://github.com/acmesh-official/acme.sh/wiki/Synology-NAS-Guide](url)

When I do the last line on creating the certificate:
./acme.sh --issue --server letsencrypt --home . -d "$CERT_DOMAIN" --dns "$CERT_DNS" --keylength 2048

I get the following error:
![error](https://github.com/user-attachments/assets/d0e2841c-84e6-4e57-84fd-30525aac2ec8)

**Comment by rioncm** [comment]
$CERT_DOMAIN should be in the format host.domain.tld in your case www.marine-captian.com or you can make it a wildcard like *.marine-captian.com to catch any host name you want to toss at the server.  if you want specific names you can specify alternates 

./acme.sh --issue --server letsencrypt --home . -d www.marine-captian.com -d myhost.marine-captian.com  --dns dns_cf --keylength 2048

If you want your host accessible with SSL at marine-captian.com use a wildcard. 

./acme.sh --issue --server letsencrypt --home . -d *.marine-captian.com  --dns dns_cf --keylength 2048

you don't have to define those as environmental variables to run acme.sh
